circle.yml

```diff
@@ -0,0 +1,82 @@
+version: 2
+workflows:
+  version: 2
+  linux-builds:
+    jobs:
+      - build-x86_64
+      - build-i686
+
+jobs:
+  build-x86_64:
+    docker:
+#      - image: ubuntu:14.04
+      - image: circleci/python:2.7
+        environment:
+          JULIA_CPU_CORES: 4
```

Comment:
I don't think this is right. Each Circle container has 2 vCPUs, and setting up 4x parallelism gives you four separate, isolated _containers_ that do the same build at the same time. That's why in my earlier PR I had the load balancer for splitting the tests between containers.